test/ngAria/ariaSpec.js

```diff
@@ -414,49 +423,24 @@ describe('$aria', function() {
   describe('aria-required', function() {
```

Comment:
There is no test verifying that when `ngRequired` evaluates to false, the `aria-required` attribute is removed.